.github/workflows/gradle.yml

```diff
@@ -94,6 +94,18 @@ jobs:
           distribution: corretto
           java-version: 21
 
+      - name: Set up NET
+        uses: actions/setup-dotnet@v4
+        with:
+            dotnet-version: 7.x
+
+      - name: Install ClangSharpPInvokeGenerator
+        run: |
+          dotnet tool install --global ClangSharpPInvokeGenerator --version 18.1.0.2
+          nuget install libclang.runtime.osx-arm64 -Version 18.1.3
+          nuget install libClangSharp.runtime.osx-arm64 -Version 18.1.3.1
+          find /Users/runner/work/doordeck-headless-sdk/doordeck-headless-sdk -name "*.dylib" -exec cp {} /Users/runner/.dotnet/tools/.store/clangsharppinvokegenerator/18.1.0.2/clangsharppinvokegenerator/18.1.0.2/tools/net8.0/any/ \;
```

Comment:
- Are you able to use relative paths so it's a bit more portable? 
- Are the dylib files checked in to source control? Because this runs before the build step 
- Do we need to look for dll (Windows)/so (Linux) files?

Reply:
* I made some changes to the paths; let me know what you think.
* The dylibs are used by `ClangSharpPInvokeGenerator`, so we only need to include the macOS-related ones